patches/tModLoader/Terraria/ModLoader/Config/ConfigManager.cs

```diff
@@ -263,6 +263,28 @@ internal static bool ModNeedsReload(Mod mod)
 		return false;
 	}
 
+	internal static bool AnyModNeedsReloadCheckOnly(IEnumerable<LocalMod> normalModsToLoad, List<ReloadRequiredExplanation> reloadRequiredExplanationEntries)
```

Comment:
Might be better to use an `out List<Mod>` or return an `IEnumerable<Mod>`. There's a lot of similar code here and in the only caller, so it's better to keep similar code together (the creation of explanations for eg)